Task 2/2.2/Moveable.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _2._2
+{
+    public abstract class Moveable : Objects
+    {
+        public Moveable(char symbol, int x, int y) : base(symbol, x, y, false) { }
+
+        public int health = 2;
+        public int damage = 1;
+        public int stepLength = 1;
+
+        public void Move()
+        {
+
+        }
+    }
+
+    public class Enemy : Moveable
+    {
+        public Enemy(int x, int y) : base('¤', x, y) { }
+
+        public override void Print()
+        {
+            Console.ForegroundColor = ConsoleColor.Red;
+            Console.Write(symbol);
+        }
```

Comment:
Okay, but it would be better if printing was implemented in a parent rather than here (you would have only one place that prints stuff)